cocos/ui/UIWidget.cpp

```diff
@@ -783,7 +783,9 @@ void Widget::propagateTouchEvent(cocos2d::ui::Widget::TouchEventType event, coco
     Widget* widgetParent = getWidgetParent();
     if (widgetParent)
     {
+        widgetParent->_hittedByCamera = _hittedByCamera;
```

Comment:
Shouldn't the previous value of `widgetParent->_hittedByCamera` be saved ? It may not be nullptr